### ISSUE TITLE
Hidden by overlayes

### ISSUE THREAD

**Issue by Marin-Kitagawa** [tridactyl/tridactyl#4537]
-   Brief description of the problem:
When I press `:` to enter the command mode or if I press `t` to open a tab from History and Bookmarks, if there are any overlays, the respective popup gets hidden behind the overlay

![image](https://user-images.githubusercontent.com/49131888/211572100-82cad1fe-15ab-473d-8012-942403df7a67.png)

-   Tridactyl version (`:version`): $1.23.0$

-   Firefox version (Top right menu > Help > About Firefox): $108.0.2 (64-\text{bit})$

-   URL of the website the bug happens on: Any URL with a stupid ad overlay as in the screenshot

-   Config (in a new tab, run `:viewconfig --user`, copy the url and paste it somewhere like gist.github.com):
```yml
╭───────────────┬───────────────────────────────────╮
│               │ ╭───────────────┬───────────────╮ │
│ update        │ │ lastchecktime │ 1673357854008 │ │
│               │ ╰───────────────┴───────────────╯ │
│ configversion │ 2.0                               │
│ smoothscroll  │ true                              │
╰───────────────┴───────────────────────────────────╯
```
-   Contents of ~/.tridactylrc or ~/.config/tridactyl/tridactylrc (if they exist):
Doesn't exist

-   Operating system: 
```yml
Edition:	Windows 11 Pro
Version:	22H2
OS build:	22621.963
Experience:	Windows Feature Experience Pack 1000.22638.1000.0
```

**Comment by bovine3dom** [comment]
I don't think we can do anything about this - we already use the maximum permissible value for how "on top" our elements should be: https://github.com/tridactyl/tridactyl/blob/e26865e4e4cbd000509bb72199ddb50bfc718a30/src/static/css/content.css#L6

If you can give me a link to a page that has an ad overlay like that I would be happy to dig a little further. But I can't suggest much other than to use an ad blocker or to modify the CSS of the page with e.g. Stylus/Stylish to make the `z-index` property of the advert popup smaller

**Comment by Marin-Kitagawa** [comment]
Oh wow. That's really big
